### ISSUE TITLE
Icon request: icon-climbing-person

### ISSUE THREAD

**Issue by Ronyo** [FortAwesome/Font-Awesome#13649]
### What can it be used for?

For applications in fitness industry it would be awesome to be able to use icons with a person climbing, lifting weights (not a single dumbbell).

### Example image (optional)

-

### Icon request checklist

- [ ] This is a single icon or matched pair (Ex: `lock` / `unlock`)
- [x] The title starts with "Icon request: " and is followed by the requested icon name (Ex: `Icon request: magical-unicorn`)
- [x] I have [searched for existing issues](https://github.com/FortAwesome/Font-Awesome/issues) and to the best of my knowledge this is not a duplicate
- [ ] I have included at least one example image if it could be helpful (optional)
- [x] The request is for a concrete object (it's harder to make an icon to represent happiness, it's easier to make a smiley face. ☺)

**Comment by Crissov** [comment]
Related Unicode emoji #222:

- [🧗](https://emojipedia.org/person-climbing/) U+1F9D7 Person Climbing

**Comment by Armitage35** [comment]
+1

**Comment by designed2use** [comment]
+1

**Comment by sandorfr** [comment]
Yes! with the growth of the rock climbing industry we definitely could use an icon for this :) I can see many good inspiration candidates in noun project https://thenounproject.com/search/?q=climbing

**Comment by electron1966** [comment]
That would be great. A complete paradigm shift.

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’re closing this issue because it’s a duplicate of or shares too much similarity with [this request.](https://github.com/FortAwesome/Font-Awesome/issues/16932)
